compiler/rustc_resolve/src/ident.rs

```diff
@@ -1422,17 +1458,27 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
         path: &[Segment],
         opt_ns: Option<Namespace>, // `None` indicates a module path in import
         parent_scope: &ParentScope<'a>,
+        used_import: UsedImport<'a>,
```

Comment:
```suggestion
        ignore_import: Option<Import<'a>>,
```
- This argument is very similar to `ignore_binding`, except here we ignore the import, so it is reasonable to name it similarly.
- Also, could move all these new arguments after the `ignore_binding` arguments (or otherwise to the last position).
- Also, the type alias doesn't seem very useful.